app/controllers/api/console.php

```diff
@@ -111,3 +115,29 @@
 
         $response->chunk('', true);
     });
+
+App::get('v1/console/resources/:resourceId')
+    ->desc('Check resource ID availability')
+    ->groups(['api', 'projects'])
```

Comment:
Also why is this part of the console service? This is more suited to the projects' service

Reply:
Rules aren't created at a project level. They should be unique across Appwrite. The data is stored in `console` DB and thus, the endpoint is also added to console controller.